power-preference/build.gradle

```diff
@@ -14,6 +14,7 @@ android {
             minifyEnabled false
         }
     }
+    namespace 'com.preference'
```

Comment:
I was unfamiliar with the namespace, nice!

I'm curious as to why it is preferred over the placing it in the AndroidManifest?

Reply:
I detailed it in the AndroidX update commit but here it is again for reference

Removed the applicationid from the manifest and converted the library and sample app to use Namespace as needed by newer gradle. Application id is recommended to match namespace. Which you can see below:

https://developer.android.com/studio/build/configure-app-module#set-namespace:~:text=application%20ID.-,Change%20the%20namespace,-In%20most%20cases